api/src/main/java/org/openmrs/module/pihmalawi/metadata/deploy/bundle/concept/MasterCardConcepts.java

```diff
@@ -19,109 +19,113 @@
+        Concept oneP = MetadataUtils.existing(Concept.class, ONE_P);
         Concept twoA = MetadataUtils.existing(Concept.class, TWO_A);
         Concept twoP = MetadataUtils.existing(Concept.class, TWO_P);
-        Concept threeA = MetadataUtils.existing(Concept.class, "657ab886-977f-11e1-8993-905e29aff6c1");
-        Concept threeP = MetadataUtils.existing(Concept.class, "657aba3e-977f-11e1-8993-905e29aff6c1");
+        Concept threeA = MetadataUtils.existing(Concept.class, THREE_A);
+        Concept threeP = MetadataUtils.existing(Concept.class, THREE_P);
         Concept fourA = MetadataUtils.existing(Concept.class, FOUR_A);
         Concept fourP = MetadataUtils.existing(Concept.class, FOUR_P);
+	Concept fiveA = MetadataUtils.existing(Concept.class, FIVE_A);
         Concept sixA = MetadataUtils.existing(Concept.class, SIX_A);
+	Concept sevenA = MetadataUtils.existing(Concept.class, SEVEN_A);
+	Concept eightA = MetadataUtils.existing(Concept.class, EIGHT_A);
         Concept nineA = MetadataUtils.existing(Concept.class, NINE_A);
         Concept nineP = MetadataUtils.existing(Concept.class, NINE_P);
         Concept tenA = MetadataUtils.existing(Concept.class, TEN_A);
         Concept elevenA = MetadataUtils.existing(Concept.class, ELEVEN_A);
-        Concept elevenP = MetadataUtils.existing(Concept.class, ELEVEN_P);
+	Concept twelveA = MetadataUtils.existing(Concept.class, TWELVE_A);
+	Concept thirteenA = MetadataUtils.existing(Concept.class, THIRTEEN_A);
+	Concept fourteenA = MetadataUtils.existing(Concept.class, FOURTEEN_A);
+	Concept fifteenA = MetadataUtils.existing(Concept.class, FIFTEEN_A);
+	Concept nonStandard = MetadataUtils.existing(Concept.class, NON_STANDARD);
 
         install(new ConceptBuilder(HOSPITALIZED_SINCE_LAST_VISIT_CONCEPT)
                 .datatype(coded)
-                .conceptClass(question)
+                .conceptClass(question)			
                 .name("65f83b26-977f-11e1-8993-905e29aff6c1", "Patient hospitalized since last visit", Locale.ENGLISH, ConceptNameType.FULLY_SPECIFIED)
-                .answers(yes, no)
+                .answers(yes, no)			
                 .build());
 
-        Concept fiveA = install(new ConceptBuilder(FIVE_A)
-                .datatype(notApplicable)
-                .conceptClass(drug)
-                .name("6628679c-977f-11e1-8993-905e29aff6c1", "5A: TDF / 3TC / EFV", Locale.ENGLISH, ConceptNameType.FULLY_SPECIFIED)
-                .build());
+	Concept elevenP = install(new ConceptBuilder(ELEVEN_P)
+                 .datatype(notApplicable)
+                 .conceptClass(drug)
+                 .name("ba40105b-673d-4fa4-9263-70ce5f228230", "11P: AZT / 3TC + LPV/r (previous AZT3TCLPV)", Locale.ENGLISH, ConceptNameType.FULLY_SPECIFIED)
+                 .build());
 
-        Concept sevenA = install(new ConceptBuilder(SEVEN_A)
+        Concept fourteenP = install(new ConceptBuilder("cf770b14-4c9d-11ea-b77f-2e728ce88125")
```

Comment:
The ConceptBuilder constructors should refer to constants, like the others, right?  It is hard to review consistency of uuids unless they refer to explicit constants.